infra/gcp/clusters/projects/k8s-infra-prow-build/prow-build/resources/build-serviceaccounts.yaml

```diff
@@ -14,3 +14,11 @@ metadata:
     iam.gke.io/gcp-service-account: gcb-builder-releng-test@k8s-staging-releng-test.iam.gserviceaccount.com
   name: gcb-builder-releng-test
   namespace: test-pods
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
```

Comment:
@cpanato this Service Account should be in https://github.com/kubernetes/k8s.io/blob/main/infra/gcp/clusters/projects/k8s-infra-prow-build-trusted/prow-build-trusted/resources/build-serviceaccounts.yaml if you want the prowjob running on `k8s-infra-prow-build-trusted`.

Reply:
done, thanks!